frontend/src/pages/Projects/Project/CloseOut/index.tsx

```diff
@@ -46,7 +46,7 @@ export const CloseOut = () => {
     default:
       content = (
         <>
-          <Notify projectId={projectId} />
+          {!userHasEditCapability && <Notify projectId={projectId} />}
```

Comment:
the wording on this might be wrong, as the user doesn't have edit capability, they would only have read

Reply:
I still don't have a great understanding of what notify should do but I thought it was only supposed to appear for users without edit capability which is what I have it doing here.

Reply:
oh wait, i didn't see the ! (not) my mistake